Documentation/project-docs/profiling-api-status.md

```diff
@@ -43,4 +43,26 @@ The notes below will help you determine what profiling APIs are safe to use. The
 ##APIs definitely known not to work yet
 ###Windows
 ###Linux
+
+* ICorProfilerInfo:
```

Comment:
Can we think of a good way to add GitHub handles or should we allow the GitHub commit history to manage that? Otherwise LGTM.